README.md

```diff
@@ -80,29 +82,25 @@ if err != nil {
 
 ### Propagating a tag map in a context
 
-To propagate a tag map to downstream methods and RPCs, add a tag map
-to the current context. NewContext will return a copy of the current context,
-and put the tag map into the returned one.
-If there is already a tag map in the current context, it will be replaced.
+You can also propagate a tag map in the current context:
```

Comment:
The code in line 89 shows the contrary though, it shows retrieving the tags map instead of propagating it.

Reply:
Did you mean to instead use `tag.New`?

Reply:
Modified the wording a bit now, it is got clearer.